src/scythe-bidder/faction-mat-combination.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import _ from "lodash";
+import { Button, Image } from "react-bootstrap";
+import { ClientSideBid, Combination, Faction, Mat, Player } from "./types";
+import { Ctx } from "boardgame.io";
+import { EventsAPI } from "boardgame.io/dist/types/src/plugins/events/events";
+
+interface Props {
+  combination: Combination;
+  moves: {
+    bid: ClientSideBid;
+  };
+  events: EventsAPI;
+  players: Array<Player>;
+  ctx: Ctx;
+  isActive: boolean;
+}
+
+class FactionMatCombination extends React.Component<Props> {
```

Comment:
Renaming this from "Combination" so that the name doesn't collide with the Combination type.